.github/ISSUE_TEMPLATE/enhancement.md

```diff
@@ -0,0 +1,16 @@
+---
+name: Enhancement Request
+about: Suggest an enhancement to tail
+
+---
+<!-- Please only use this template for submitting enhancement requests -->
+
+
+**What would you like to be added**:
+
+**Why is this needed**:
```

Comment:
I used the one in kubernetes/kubernetes https://github.com/kubernetes/kubernetes/blob/master/.github/ISSUE_TEMPLATE/enhancement.md. Should this be "User story" instead?

Reply:
I think this is fine as-is, one can just put his or her user story below it